docs/triaging/safe-responses.md

```diff
@@ -24,19 +24,19 @@ This will be rejected for now and we can reconsider if we open a programming / c
 
 ### Knowledge not Skill
 
-Given the content of the file, it seems that this would likely fall under knowledge, which we are currently accepting contributions for under different [guidlines](https://github.com/instructlab/taxonomy?tab=readme-ov-file#getting-started-with-knowledge-contributions).
```

Comment:
I think the idea of having these urls here was the idea of being able to copy pasta this into the Issue/Slack message. If we make them relative, then will that work?

Reply:
It should. We do it with lines 9, 15, and 21...